pattern_library/views.py

```diff
@@ -94,3 +97,17 @@ def get(self, request, pattern_template_name=None):
             return self.render_to_response(context)
 
         return HttpResponse(rendered_pattern)
+
+
+@csrf_exempt
+def render_pattern_api(request):
+    data = json.loads(request.body.decode("utf-8"))
+    template_name = data["template_name"]
+    config = data["config"]
```

Comment:
Question: Is the intent here to always have to pass in the whole `config` object, or is there any desire to just pass in overrides and merge the context sent to this endpoint with the YAML config?

Reply:
I think this endpoint is easier to reason about if you pass the whole config. My assumption is that if you’re using those endpoints, then you might not use YAML configs at all?